projects/Mallard/src/theme/typography.ts

```diff
@@ -264,7 +264,7 @@ const scale = {
             },
             [Breakpoints.phone]: {
                 fontSize: 60,
-                lineHeight: 66,
+                lineHeight: 58,
```

Comment:
Can we remove this type change from this PR please? These values are used in multiple places and as such can have unintended consequences that need to be tested thoroughly across devices.